lib/usb-connection.js

```
@@ -23,7 +29,7 @@ try {
   // var VENDOR_REQ_IN  = usb.LIBUSB_REQUEST_TYPE_VENDOR | usb.LIBUSB_RECIPIENT_DEVICE | usb.LIBUSB_ENDPOINT_IN;
 } catch (e) {
   haveusb = false;
-  logs.err('WARNING: No usb controller found on this system.');
+  log.error('WARNING: No usb controller found on this system.');
 }
```

That's a fail fast error, I don't think we need to do anything